commit 7dc5ba39e597a6d761de45e5c9de24fa0c826557
Author: Alexey Dobriyan <adobriyan@gmail.com>
Date:   Sat Aug 19 12:57:51 2017 +0300

    genirq/ipi: Fixup checks against nr_cpu_ids
    
    commit 8fbbe2d7cc478d1544f41f2271787c993c23a4f6 upstream.
    
    Valid CPU ids are [0, nr_cpu_ids-1] inclusive.
    
    Fixes: 3b8e29a82dd1 ("genirq: Implement ipi_send_mask/single()")
    Fixes: f9bce791ae2a ("genirq: Add a new function to get IPI reverse mapping")
    Signed-off-by: Alexey Dobriyan <adobriyan@gmail.com>
    Signed-off-by: Thomas Gleixner <tglx@linutronix.de>
    Link: http://lkml.kernel.org/r/20170819095751.GB27864@avx2
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1654_7dc5ba39e597a6d761de45e5c9de24fa0c826557.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1654_7dc5ba39e597a6d761de45e5c9de24fa0c826557.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
